app/components/listing/ServiceDetails.tsx

```diff
@@ -16,14 +17,10 @@ export const ServiceDetails = ({ service }: { service: Service }) => {
       id={`service-${service.id}`}
       data-cy="service-list-item"
     >
-      {/* <div className="service--meta disabled-feature">
-        <p><ServiceCategory category={service.category} /></p>
-        <p>updated {service.updated_at}</p>
-      </div> */}
       <h2 className="service--header">
-        <a href={`/services/${service.id}`} translate="no">
+        <Link to={{ pathname: `/services/${service.id}` }} translate="no">
```

Comment:
During my GA debugging I found that clicking on this particular anchor link would trigger a refetch of the entire document; whereas using `Link to` does not. It did not solve our GA problem but is a good improvement nonetheless.